README.md

```diff
@@ -1,3 +1,5 @@
+### Dobriy den everyody. Im Vasyl, from Spain.
```

Comment:
Hey, good work, can you pls only move your greeting under the link to the discord pls?